SPECS-EXTENDED/python-evdev/python-evdev.signatures.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
It tripped me at first .. but its just `git` removing and re-adding this file, since it was moved from extended -> core

Reply:
Correct!